app/components/message-input/template.hbs

```diff
@@ -1,7 +1,9 @@
-<form {{action onSend on="submit"}}>
-  {{channel-input-field value=message
-                        placeholder="Type a message and hit enter"
-                        name="chat-message"
-                        id="message-field"
-                        autocomplete="off"}}
-</form>
+<section id="new-message" {{did-insert this.didInsertNode}} {{will-destroy this.willDestroyNode}}>
```

Comment:
Same as above. I would suggest to use more descriptive method names.

Reply:
Same as above. Is this something that you wouldn't merge without?